app/src/config.json

```diff
@@ -2,8 +2,8 @@
   "name": "Cosmos Voyager",
   "default_network": "local-testnet",
   "denoms": ["stake", "photino"],
-  "node_lcd": "http://localhost:8080",
-  "node_rpc": "http://localhost:26657",
+  "lcd": "http://localhost:8080",
```

Comment:
not intended to block this PR, but aren't we supposed to be getting rid of `lcd` in favour of `stargate`? did we decide that already?

Reply:
not globally, although we use `gaia-lite` more than `lcd` rn

Reply:
would be nice to pick one.

Reply:
vote for `stargate` :D

Reply:
well, i would pick stargate over LCD or something like that. but it might be worth calling it `gaiaAPI` or something more obvious like that. anyways, for another issue.

Reply:
created an issue for it.